guacamole/src/main/frontend/src/app/rest/services/tunnelService.js

```diff
@@ -96,20 +96,18 @@ angular.module('rest').factory('tunnelService', ['$injector',
      *     success.
      */
     service.getProtocol = function getProtocol(tunnel) {
-
-        // Build HTTP parameters set
-        var httpParameters = {
-            token : authenticationService.getCurrentToken()
-        };
-
+        // GUACAMOLE-956: replace http parameter with header | START
         // Retrieve the protocol details of the specified tunnel
         return requestService({
             method  : 'GET',
             url     : 'api/session/tunnels/' + encodeURIComponent(tunnel)
                         + '/protocol',
-            params  : httpParameters
-        });
+            headers: {
+                'Guacamole-Token': authenticationService.getCurrentToken(),
+            }
 
+        });
+        // GUACAMOLE-956: replace http parameter with header | END
```

Comment:
Please don't bracket your changes in start/end comments noting the JIRA issue. It's better that the changes be just the changes themselves, plus any necessary documentation.